pySMART/device.py

```diff
@@ -1028,6 +1034,13 @@ class members, including the SMART attribute table and self-test log.
                     for line in _stdout.split('\n'):
                         if 'power on time' in line:
                             self.diags['Power_On_Hours'] = line.split(':')[1].split(' ')[1]
+        # map temperature
+        if self.temperature is None:
+            # in this case the disk is probably ata
+            try:
+                self.temperature = int(self.attributes[194].raw)
+            except:
```

Comment:
I would go here with `except ValueError` rather than naked `except`. For example in scenario when, for whatever reason, 'attributes' does not have index 194 you will mask the exception.